openassessment/staffgrader/staff_grader_mixin.py

```diff
@@ -194,6 +194,39 @@ def list_staff_workflows(self, data, suffix=''):  # pylint: disable=unused-argum
                 log.exception("Failed to serialize workflow %d: %s", staff_workflow.id, str(e), exc_info=True)
         return result
 
+
+
+    @XBlock.json_handler
+    @require_course_staff("STUDENT_GRADE")
+    def list_assessments_grades(self, data, suffix=''):  # pylint: disable=unused-argument
```

Comment:
why `list_assessment_grades` ? if we're almost returning all assessment fields: https://github.com/openedx/edx-ora2/blob/77c2dbe104b032f716a1dd5fb21ae60761a859f9/openassessment/assessment/models/base.py#L409-L427

Reply:
changed to `list_assessments`